libs/node-api/proto/nillion/health/v1/health.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
```

Comment:
Is this file actually needed? You're using `tonic_health` which comes with this prepackaged, is this here so that reflection includes it? If that's the case I think you should be able to include that directly, see [docs](https://docs.rs/tonic-health/latest/tonic_health/pb/constant.FILE_DESCRIPTOR_SET.html).

Otherwise if this is actually needed, the path should match the package (e.g. `nillion/health/v1/health` vs `grpc.health.v1`

Reply:
> If that's the case I think you should be able to include that directly, see [docs](https://docs.rs/tonic-health/latest/tonic_health/pb/constant.FILE_DESCRIPTOR_SET.html).

you are right, its public. thanks for checking!